src/App/Hovedside/TjenesteBoksContainer/Arbeidstreningboks/Arbeidstreningboks.tsx

```diff
@@ -1,17 +1,73 @@
-import React, { FunctionComponent } from "react";
+import React, {
+  FunctionComponent,
+  useContext,
+  useEffect,
+  useState
+} from "react";
 import arbeidstreningikon from "./arbeidstreningikon.svg";
 import Lenkepanel from "nav-frontend-lenkepanel";
 import "./Arbeidstreningboks.less";
 
 import TjenesteBoksBanner from "../TjenesteBoksBanner/TjenesteBoksBanner";
 import { syfoLink } from "../../../../lenker";
+import { OrganisasjonsDetaljerContext } from "../../../../OrganisasjonDetaljerProvider";
+import { Arbeidsavtale } from "../../../../api/dnaApi";
 
 interface Props {
   varseltekst?: string;
   className?: string;
 }
 
 const Arbeidstreningboks: FunctionComponent<Props> = props => {
+  const { arbeidsavtaler } = useContext(OrganisasjonsDetaljerContext);
+  const [
+    antallKlareStillingsannonserTekst,
+    setantallKlareStillingsannonserTekst
+  ] = useState("");
+  const [antallTilGodkjenningTekst, setantallTilGodkjenningTekst] = useState(
+    ""
+  );
+  const [antallUnderArbeidTekst, setantallUnderArbeidTekst] = useState("");
+
+  const LagTekstBasertPaAntall = (antall: string) => {
+    if (antall === "1") {
+      return " arbeidsavtale ";
+    }
+    return " arbeidsavtaler ";
+  };
+
+  useEffect(() => {
+    const KlareForOppstartArbeidsavtaler: Array<
```

Comment:
Du kan typesette denne som `Arbeidsavtale[]` i stedet for `Array<Arbeidsavtale>` - det er vanligere, og i dette tilfellet slipper man å bruke en del vertikal plass :)